.github/workflows/pr.yml

```diff
@@ -34,10 +34,10 @@ jobs:
           export PYTHONHASHSEED=42
           git checkout -b branch
           git fetch --unshallow origin main
-          cmd="ansible-lint --parseable --nocolor --profile=shared"
+          cmd="ansible-lint --parseable --force-color --profile=shared"
```

Comment:
Forcing color might need sanitizing the ansi color codes when printing the results, we remove them with `sed -r 's/\x1B\[[0-9]{1,2}[mGK]//g'` in the sanity checks. Perhaps lines 49 and 53?

Reply:
Added the filtering and modified slightly the regex to match the shading used by ansible-lint